ledgerLiveManifest.json

```diff
@@ -1,28 +0,0 @@
-{
-  "id": "olympus",
-  "name": "Olympus",
-  "url": "http://localhost:3000/app/dapp-browser?dappName=Olympus&nanoApp=Olympus&url=http%3A%2F%2Flocalhost%3A3001",
-  "homepageUrl": "https://olympusdao.finance/",
-  "icon": "https://res.cloudinary.com/appleseed/image/upload/v1634164239/ohm-gray_xsbilr.png",
-  "platform": "all",
-  "apiVersion": "0.0.1",
-  "manifestVersion": "1",
-  "branch": "stable",
-  "categories": [
-    "defi",
-    "staking",
-    "bonds"
-  ],
-  "currencies": [
-    "ethereum"
-  ],
-  "content": {
-    "shortDescription": "The decentralized reserve infrastructure that's community-owned and liquidity protected.",
-    "description": "Olympus is building a community-owned decentralized financial infrastructure to bring more stability and transparency for the world."
-  },
-  "permissions": [],
-  "domains": [
-    "https://*",
-    "http://*"
-  ]
-}
```

Comment:
@appleseed-iii - just double checking- is this used or pointed to by ledger outside of our app?